catalog/mapbiomas-public/projects_mapbiomas-public_assets_brazil_lulc_v1.jsonnet

```diff
@@ -0,0 +1,223 @@
+
+  version: version,
+
+  description: |||
+    MapBiomas Land Use and Land Cover (LULC) dataset for Brazil is produced annually by the MapBiomas Project using Landsat satellite imagery and machine learning classification techniques.
+    The dataset provides consistent, thematically detailed maps at 30-meter resolution, covering multiple decades and updated each year.
+
+    Each image in the collection contains annual land cover classifications with pixel values representing categorical land cover classes, such as forest, agriculture, pasture, water bodies, and urban areas.
+    The classification legend is standardized and maintained across all years, enabling multi-temporal analyses of land use change, deforestation, reforestation, and other landscape dynamics.
+
+    The classification is based on Landsat 5 (TM), Landsat 7 (ETM+), Landsat 8 and 9 (OLI/TIRS) surface reflectance data, preprocessed and mosaicked to generate cloud-free composites for each year.
+    The classification process includes automated decision trees trained with reference samples, along with manual validation by regional experts.
+
+    Each image contains one band named "classification", representing the land cover classification for that specific year, version and collection id.
+
+    For more information, classification legend, methodology, and accuracy assessments, visit the [MapBiomas website](https://mapbiomas.org/en).
+
+    It is recommended to consult the classification legend to interpret the class values.
+    Note that MapBiomas provides discrete classification maps, not probabilities.
+    These maps are most appropriate for map transitions, time series analysis, and land policy monitoring applications.
+
+  |||,
+
+  'gee:categories': [
+    'landuse-landcover'
+  ],
+
+  keywords: [
+    'landsat_derived'
+  ],
+
+  providers: [
+    ee.producer_provider('MapBiomas', 'https://mapbiomas.org'),
+    ee.host_provider(self_ee_catalog_url),
+  ],
+
+  extent: ee.extent(
+    -74.00,
+    -34.00,
+    -34.70,
+    5.45,
+    '1985-01-01T00:00:00Z',
+    '2024-12-31T00:00:00Z'
+  ),
+
+  summaries: {
+    'gee:schema': [
+      {
+        name: 'year',
+        description: |||
+          Year of the land use and land cover classification.
+        |||,
+        type: ee_const.var_type.int
+      },
+      {
+        name: 'version',
+        description: |||
+          Version of the land use and land cover classification.
+        |||,
+        type: ee_const.var_type.string
+      },
+      {
+        name: 'collection_id',
+        description: |||
+          Collection identifier of the land use and land cover classification.
+        |||,
+        type: ee_const.var_type.double
+      },
+    ],
+    gsd: [30],
+
+    'eo:bands': [
+      {
+        name: 'classification',
+        description: 'Land use and land cover classification, with integer values corresponding to MapBiomas legend classes.',
```

Comment:
One last comment: could you specify the classes here similar to https://github.com/google/earthengine-catalog/blob/main/catalog/AAFC/AAFC_ACI.jsonnet#L79 ?